### ISSUE TITLE
how to sync from latest on devnet?

### ISSUE THREAD

**Issue by sylviahuang123** [multiversx/mx-chain-scripts#16]
My observing squad node is syncing very slow on devnet, and it's start from 1, I just want to start from latest, how to configure? I've tried both /node `-operation-mode snapshotless-observer` and `-full-archive false`, but none works, can you please show me the detail operation? Thank you.

**Comment by bogdan-rosianu** [comment]
As discussed on Discord, if removing the `db` directory and only syncing the most recent chain epoch does not help with the sync time, then probably the machine used is under specs

**Comment by sylviahuang123** [comment]
I'm talking about the syncing from latest epoch, all method are not work.

**Comment by sylviahuang123** [comment]
yes, I'm following [these](https://docs.multiversx.com/integrators/observing-squad) steps cleanup/remove_db many times
`cat elrond-nodes/node-0/config/prefs.toml`
`FullArchive = false`
<img width="1193" alt="image" src="https://github.com/multiversx/mx-chain-scripts/assets/100252671/9dc271ff-6d33-49a1-9901-dddccc44d2d3">
let me know if you need more.
